csharp/test/Ice/interceptor/Client.cs

```diff
@@ -79,7 +79,7 @@ private void Run(IMyObjectPrx prx, Interceptor interceptor)
 
             output.Write("testing binary context... ");
             output.Flush();
-            bool ice2 = Communicator()!.DefaultProtocol != Protocol.Ice1;
+            bool ice2 = GetTestProtocol() != Protocol.Ice1;
```

Comment:
This is weird. `bool ice2 = helper.Protocol == Protocol.Ice2` looks more reasonable!